routes/routes.js

```diff
@@ -9,4 +11,22 @@ router.get('/cheatsheet', (req,res) => {
     res.render('cheatsheet');
 })
 
+router.post('/user/answers/question/:questionNumber', (req, res) => {
+    if(req.body.id) {
+		return res.status(400).json({'message': 'ID should not be provided'});
+    }    
+    UserAnswers.create({
+        user_id: req.body.userId,
```

Comment:
Since neither Auth0 or fake auth is yet implemented, can you add a line above this that calls `User.findOrCreate({ where: { id: req.body.userId } })`...otherwise the insert won't work because the user will never exist.

It should have a comment saying that it's just needed until https://github.com/Techtonica/keyboard-shortcuts-practice/issues/74 is implemented.

Thanks!

Reply:
Yes! You're right, I wanted to use the Sequelize errors but this is a more explicit way to validated de fk :blush:  I already fixed in my recent commit.